.github/workflows/acs.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           cat $FILE_PATH | psql $EDM_DATA -v TABLE_NAME=$TABLE_NAME -f pipelines/create_acs.sql
 
       - name: upload to s3
+        if: github.ref == 'refs/heads/dev'
```

Comment:
Is this supposed to be included?

Reply:
my bad, seems like I accidentally involve others' commit

Reply:
Okay, I branched off from dev. I will modify it.